.github/workflows/check-if-migration-is-required.yml

```diff
@@ -47,3 +47,4 @@ jobs:
         run: |
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
+          sentry django makemigrations --check || echo >&2 "::error::Error: You have not commited the migration_lockfile.txt"
```

Comment:
Note that `migrations_lockfile.txt` might not be updated for 2 reasons:
 - They forgot to commit the lockfile that they produced
 - They manually added a new migration without using `makemigrations`, and so the change wasn't generated.

Might be worth calling out both options here.